consensus/src/network_interface.rs

```diff
@@ -47,6 +47,13 @@ pub enum ConsensusMsg {
     /// VoteMsg is the struct that is ultimately sent by the voter in response for receiving a
     /// proposal.
     VoteMsg(Box<VoteMsg>),
+    /// CommitProposal is the struct that is sent by the validator after execution to propose
+    /// on the committed state hash root.
+    CommitProposalMsg(Box<CommitProposal>),
+    /// CommitDecision is the struct that is sent by the validator after collecting no fewer
+    /// than 2f + 1 signatures on the commit proposal. This part is not on the critical path, but
+    /// it can save slow machines to quickly confirm the execution result.
+    CommitDecisionMsg(Box<CommitDecision>),
```

Comment:
I'd prefer to embed this in SyncInfo, maybe we can leave CommitDecision for later and only add CommitProposal for now

Reply:
Agree. We can make this change later